src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -277,6 +390,27 @@ void IslandWindow::SetContent(winrt::Windows::UI::Xaml::UIElement content)
     _rootGrid.Children().Append(content);
 }
 
+// Method Description:
+// - Gets the difference between window and client area size.
+// Arguments:
+// - dpi: dpi of a monitor on which the window is placed
+// Return Value
+// - The size difference
+SIZE IslandWindow::GetTotalNonClientExclusiveSize(const UINT dpi) const noexcept
+{
+    RECT islandFrame{};
+
+    // If we failed to get the correct window size for whatever reason, log
+    // the error and go on. We'll use whatever the control proposed as the
+    // size of our window, which will be at least close.
+    LOG_IF_WIN32_BOOL_FALSE(AdjustWindowRectExForDpi(&islandFrame, WS_OVERLAPPEDWINDOW, false, 0, dpi));

```

Comment:
This guy should also be using something more tightly bound to the exact flags that were used to CreateWindow that HWND instead of stabbling WS_OVERLAPPEDWINDOW in there as another constant declaration.

Reply:
That's #3181

Reply:
OK, again, sorry. Will go read it.